composer.json

```diff
@@ -29,12 +29,14 @@
     "require": {
         "php": ">=5.3.3",
         "marc-mabe/php-enum":"^2.0 || ^3.0 || ^4.0",
-        "icecave/parity": "1.0.0"
+        "icecave/parity": ">=1.0.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.2.20 || ~2.19.0",
         "json-schema/json-schema-test-suite": "1.2.0",
-        "phpunit/phpunit": "^4.8.35"
+        "phpunit/phpunit": "^4.8.36 || ^5.7.27 || ^6.5.14 || ^7.5.20 || ^8.5.29 || ^9.5.24",
+        "sanmai/phpunit-legacy-adapter": "^6.4 || ^8.2.1",
```

Comment:
There's a certain bias since I'm the author of this package. In my defense, it is [used by several high-profile packages with millions of installs](https://packagist.org/packages/sanmai/phpunit-legacy-adapter/dependents?order_by=downloads).